ci/common/test-pyclaragenomics.sh

```diff
@@ -19,6 +19,7 @@ cd $PYCLARAGENOMICS_DIR
 
 #Install external dependencies.
 python -m pip install --ignore-installed -r requirements-base.txt && python -m pip install -r requirements-macs2.txt
+apt install hdf5-tools
```

Comment:
this works in CI without using `sudo`?

Reply:
this was not supposed to be here, removed it. Added to the dockerfile.